bigtable/tests/unit/test_row_data.py

```diff
@@ -189,117 +189,85 @@ def test_row_key_getter(self):
 
 
 class TestPartialRowsData(unittest.TestCase):
+    @staticmethod
+    def _get_yield_target_class():
+        from google.cloud.bigtable.row_data import YieldRowsData
+
+        return YieldRowsData
 
     @staticmethod
-    def _get_target_class():
+    def _get_partial_target_class():
```

Comment:
Please revert this name change. The test `TestPartialRowsData` is meant to test the `PartialRowsData` class, hence this method returns the "class under test".

You should add a **new** `TestYieldRowsData` and test the newly factored out class there.